notebooks/MAST/astroqueryMAST/beginner_search_astroqueryMAST/beginner_search_astroqueryMAST.ipynb

```diff
@@ -0,0 +1,395 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"top\"></a>\n",
+    "# MAST API Tutorial - Astroquery API\n",
+    "\n",
+    "## Introduction and Goals:\n",
+    "This is a beginner tutorial on accessing the MAST database using the Astroquery API. By the end of this tutorial, you will:\n",
+    "\n",
+    "* Understand how to search for observations hosted on the MAST archive\n",
+    "* Download data products corresponding to your observations of interest\n",
+    "* Create a visual display of the downloaded data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Table of Contents\n",
+    "\n",
+    "* [Imports](#Imports)\n",
+    "\n",
+    "\n",
+    "* [Three ways to search for MAST observations](#Three-Ways-to-Search-for-MAST-Observations)\n",
+    "    - [By Region](#1.-By-Region)\n",
+    "    - [By Object Name](#2.-By-Object-Name)\n",
+    "    - [By Criteria](#crit)\n",
+    "    \n",
+    "    \n",
+    "* [Getting Associated Data Products](#Getting-Associated-Data-Products)\n",
+    "    - [Performing a Product Query](#Performing-a-Product-Query)\n",
+    "    - [Filtering Data Products](#Filtering-the-Data-Products)\n",
+    "    \n",
+    "    \n",
+    "* [Downloading Products](#Downloading-Products)\n",
+    "\n",
+    "\n",
+    "* [Displaying Data](#Displaying-Data)\n",
+    "\n",
+    "\n",
+    "* [Further Reading](#Further-Reading)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The only import we need for the MAST API is `astroquery.mast`. \n",
+    "\n",
+    "`astropy` and `matplotlib` are for opening and plotting the data, respectively."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astroquery.mast import Observations\n",
+    "\n",
+    "from astropy.io import fits\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.colors import SymLogNorm\n",
+    "%matplotlib inline "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Three Ways to Search for MAST Observations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. By Region\n",
+    "You can use `query_region` to find objects based on coordinates and radius.  \n",
+    "The coordinates can be given as a string or `astropy.coordinates` object, and the radius as a string or float object. If no radius is specified, the default is 0.2 degrees."
```

Comment:
I think we should have two cells here, one showing how to search for this coordinate with a stirng and no specified radius (what you have now), and another cell showing how to specify the same coordinate but passed as an astropy.coordinates object and by specifying a serach radius (somethng measured in arcseconds to show how that works too, maybe 5 arsecond radius).